packages/common/src/request.ts

```diff
@@ -50,16 +50,26 @@ export function hubApiRequest(
     ...{ "Content-Type": "application/json" },
     ...options.headers
   };
-  // TODO: build query params/body based on requestOptions.params?
+  // build query params/body based on requestOptions.params
+  let query;
+  let body;
+  if (options.httpMethod === "GET") {
+    // pass params in query string
+    query = options.params;
+  } else {
+    // pass params in request body
+    body = JSON.stringify(options.params);
```

Comment:
@rgwozdz can you verify that for POST operations the API expects the body to be stringified JSON?

Reply:
the hub api expects json, so this is stringifying a javascript object into json. in short this should be fine.